src/contract/mod.rs

```diff
@@ -362,7 +362,7 @@ mod tests {
         };
 
         // then
-        transport.assert_request("eth_estimateGas", &["{\"data\":\"0x06fdde03\",\"from\":\"0x0000000000000000000000000000000000000005\",\"to\":\"0x0000000000000000000000000000000000000001\"}".into(), "\"latest\"".into()]);
+        transport.assert_request("eth_estimateGas", &["{\"data\":\"0x06fdde03\",\"from\":\"0x0000000000000000000000000000000000000005\",\"to\":\"0x0000000000000000000000000000000000000001\"}".into()]);
```

Comment:
Can you add a test with the block number set as well?

Reply:
Added in `Eth` transport tests.